frontend/public/components/masthead-toolbar.jsx

```diff
@@ -264,6 +265,11 @@ class MastheadToolbarContents_ extends React.Component {
             externalLink: true,
             href: getOCMLink(clusterID),
             image: <img src={redhatLogoImg} alt="" />,
+            callback: () => {
+              fireTelemetryEvent('Launcher Menu Accessed', {
+                item: 'OpenShift Cluster Manager',
```

Comment:
Since we want to also capture an ID for the ConsoleLink contributed items, let's duplicate the constant value here as both the ID and the name.
```suggestion
                id: 'OpenShift Cluster Manager',
                name: 'OpenShift Cluster Manager',
```